Backend/app/rider-platform/rider-app/Main/spec/API/MultiModal.yaml

```
@@ -192,6 +199,15 @@ apis:
         journeyId: Id Journey
       response:
         type: JourneyBookingPaymentStatus
+  - POST:
+      endpoint : /multimodal/{journeyId}/payment/updateOrder
```

YAMLlint (1.35.1)

[warning] too many spaces before colon

(colons)